hibernate-core/src/main/java/org/hibernate/dialect/H2Dialect.java

```diff
@@ -122,7 +122,7 @@ public H2Dialect() {
 		registerColumnType( Types.BIT, "boolean" );
 		registerColumnType( Types.CHAR, "char($l)" );
 		registerColumnType( Types.DATE, "date" );
-		registerColumnType( Types.DECIMAL, buildId >= 201 ? "numeric($p,$s)" : "decimal($p,$s)" );
+		registerColumnType( Types.DECIMAL, "decimal($p,$s)" );
```

Comment:
I'm confused. Does this mean a part of #3405 was not necessary at all?
Or do we have the same problem as here: https://github.com/hibernate/hibernate-orm/pull/3519#discussion_r483961535?

Reply:
Well that was the result of the long discussion we just had, wasn't it? I mean, honestly I found it pretty hard to understand what he was saying, but after much back and forth it seems to me we agreed that the workaround wasn't required at all from .200 on.